### ISSUE TITLE
Questions about evaluation

### ISSUE THREAD

**Issue by GarlicBa** [facebookresearch/vggt#282]
Hi, thank you for your amazing work on VGGT!
I'm relatively new to the 3D reconstruction field, and I'm currently attempting to reproduce the depth estimation and point cloud evaluation results, and I would be very grateful if you could help clarify a few questions I encountered during the process：

1. When reproducing the depth estimation results, I found that VGGT's predicted depth map values are in [0,1] (since the training data is normalized). In [ #208](https://github.com/facebookresearch/VGGT/issues/208), you kindly mentioned aligning the prediction to ground truth using GT depth maps and shared related code. However, as far as I know, the DTU test set does not directly provide ground truth depth maps. I have checked data resources such as GeoNVSNet and couldn't find them either. Am I missing something? Or is there any other community resource for this?Additionally, after aligning the predicted depth map  and fusing the point cloud, do I still need to further align the fused point cloud to the GT point cloud for evaluation?

2. In your ETH3D evaluation, you mentioned "The results are reported after filtering out invalid points using the official masks." What exactly are these official masks? Why is this mask filtering necessary, and what is its purpose?

Thank you very much for your time!

**Comment by dfcl2209** [comment]
Hi, i meet the same questions with you,  i do not konw how to evaluate the ETH3D dataset too，even jytime had given the method，have you ever solve this  problem？
